### ISSUE TITLE
CMRI data is now available via OASIS. Can you please add Capacity Testing data set WEIM_RSE_CAPACITY_TEST?

### ISSUE THREAD

**Issue by ThinkCode** [kmax12/gridstatus#250]
WEIM_RSE_CAPACITY_TEST is the dataset we are looking for now that CMRI source is no longer providing the same data.

**Comment by kmax12** [comment]
are you speaking about this dataset? 

<img width="905" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/a2f408c0-2ef5-4612-b9de-939a99d33ca4">
<img width="1531" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/51ef885e-04c7-4085-b77f-f0201cefbcd7">

**Comment by ThinkCode** [comment]
Yeah, that is it! CMRI is no longer providing that info and OASIS has it as shown in that screenshot. Is it doable? Two other sources stopped showing up in CMRI. Hunting the dataset name.
